### DISCUSSION TITLE
Minio Gateway S3 cn-north-1 not working

### DISCUSSION THREAD

**Discussion by sofsms** [minio/minio#16331] [Q&A] [answered]
When I add the endpoint https://s3.cn-north-1.amazonaws.cn to minio deployment the pod start working but the access to bucket is denied. The role and policy that is being used are right because I'm using the same for other regions (eu-central-1) and it is working.

My question is: Can Minio gateway s3 be used for cn-north-1 region?

If yes, why it is not working when I define that endpoint?

Release verion used: RELEASE.2022-02-01T18-00-14Z

**Comment by harshavardhana** [comment, marked as answer]
We do not support gateway anymore @sofsms
